streaming/stream_transfer_task.cc

```diff
@@ -237,21 +237,28 @@ future<> stream_transfer_task::execute() {
     }).then([this, id, plan_id] {
         _mutation_done_sent = true;
         sslog.debug("[Stream #{}] GOT STREAM_MUTATION_DONE Reply from {}", plan_id, id.addr);
-    }).handle_exception([this, plan_id, cf_id, id] (std::exception_ptr ep) {
+    }).handle_exception(coroutine::lambda([this, plan_id, cf_id, id] (std::exception_ptr ep) -> future<> {
         // If the table is dropped during streaming, we can ignore the
         // errors and make the stream successful. This allows user to
         // drop tables during node operations like decommission or
         // bootstrap.
-        if (!session->manager().db().column_family_exists(cf_id)) {
-            sslog.warn("[Stream #{}] Ignore the table with table_id {} which is dropped during streaming: {}", plan_id, cf_id, ep);
-            if (!_mutation_done_sent) {
-                return session->manager().ms().send_stream_mutation_done(id, plan_id, _ranges, cf_id, session->dst_cpu_id);
-            }
-            return make_ready_future<>();
+        //
+        // the db table metadata on different shards are not necessarily in
```

Comment:
Nit: 
s/the/The
s/so/So

Reply:
well, i don't type capital characters unless in formal text.

Reply:
I am not sure what is formal to you ;) But the comments are for readers not just you. It's better to make it better.

Reply:
"formal" means user-facing text.

Reply:
Like we even have a spell check hook for each PR recently in the CI,  it is best we make it better for people who we see the code / comments as well.

Reply:
thank you for the reminder. should be better now.